### DISCUSSION TITLE
Is it actually possible to drag one GridItem into another GridItem within a GridLayout

### DISCUSSION THREAD

**Discussion by skusebauch** [react-grid-layout/react-grid-layout#1614] [Q&A]
I am wondering whether there is an already well know solution to drag one `gridItem` into another within one `gridLayout`?

Pretty much similar to the solution which already exists on a `gridLayout` level - that you can drop an Item outside a `gridLayout`

```
// Calls when an element has been dropped into the grid from outside.
onDrop: (layout: Layout, item: ?LayoutItem, e: Event) => void,
```

We have already implemented an advanced `gridLayout` with a lot of nested levels, and it would be great if there is an easy solution to drag in this picture `test1` into `test2`:
![image](https://user-images.githubusercontent.com/57953090/142638976-9cf02ca0-ad63-4bb5-8f64-b2ee8132da19.png)

Most probably I have to make a workaround and wrap each Item into its own `gridLayout`? 

I would love to hear any ideas/experiences of this requirement 😊 Thank you!

**Comment by rhombergalessandro** [comment]
Hey @skusebauch, unfortunately you did not receive an answer to your question, but I was asking myself the same question and was wondering if you have any solution or experience with this to share? 👍

**Comment by qld-cf** [comment]
same question
